docs/patterns/components/LoaderOverlay/index.js

```diff
@@ -50,6 +45,15 @@ const LoaderOverlayDemo = () => {
         variant: {
           type: 'string',
           description: 'Loader variant, can be "small" or "large".'
+        },
+        focus: {
+          type: 'boolean',
+          description: 'whether or not to focus the loader on initial render'
+        },
+        loaderRef: {
+          type: 'function',
+          description:
+            'DEPRECATED - replace with "focus" prop - optional ref function to move focus to loader on mount'
```

Comment:
While backwards compatibility will be supported with this PR, still think it should be encouraged to make use of the `focus` prop instead.

Reply:
A couple of thoughts here...
- needing an handle on the loader (`loaderRef`) goes beyond just `focus`
- There are scenarios, like loading state on page load, where one might not want to focus the loader

I am just bringing this up to ensure we are cautious with how we document this

Reply:
> While backwards compatibility will be supported with this PR, still think it should be encouraged to make use of the `focus` prop instead.

That's a good point - think just leave loaderRef as a more ambiguous statement for now, then?